tests/console/systemd_testsuite.pm

```diff
@@ -28,6 +29,12 @@ sub run {
     my $package_name = 'systemd-v$systemd_version-testsuite';
     assert_script_run 'zypper -n se -sx ' . $package_name;
     zypper_call 'in ' . $package_name;
+    if (is_leap('16.0+') || is_tumbleweed || is_sle && sle_version_at_least('16')) {
```

Comment:
I see that you using new approach with '+' for is_leap , why you not doing the same for is_sle ?

Reply:
I was using the old one, but it failed because it was removed for leap. "sle_version_at_least" still exist so I didn't notice that there was also a new approach for is_sle.